code/modules/unit_tests/map_tests.dm

```diff
@@ -19,48 +19,6 @@
 	LAZYADD(fail_reasons, "[T.x],[T.y],[T.z]: [reason]")
 	failure_count++
 
-/**
-  * Check atmos pipes are not routed under unary devices such as vents and
-  * scrubbers.
-  */
-/datum/map_per_tile_test/pipe_vent_checker
-	var/list/pipe_roots = list(
-		/obj/machinery/atmospherics/pipe/manifold/hidden/supply,
-		/obj/machinery/atmospherics/pipe/simple/hidden/scrubbers
-	)
-	var/list/unary_roots = list(
-		/obj/machinery/atmospherics/unary
-	)
-
-/datum/map_per_tile_test/pipe_vent_checker/CheckTile(turf/T)
-	var/has_pipe = FALSE
-	var/has_unary = FALSE
-
-	for(var/pipe_root in pipe_roots)
-		if(locate(pipe_root) in T.contents)
-			has_pipe = TRUE
-
-	if(locate(/obj/machinery/atmospherics/unary) in T.contents)
-		has_unary = TRUE
-
-	if(has_pipe && has_unary)
-		Fail(T, "pipe on same tile as vent or scrubber")
-
-/**
-  * Check that only one cable node exists on a tile.
-  */
-/datum/map_per_tile_test/cable_node_checker
-
-/datum/map_per_tile_test/cable_node_checker/CheckTile(turf/T)
-	var/center_nodes = 0
-
-	for(var/obj/structure/cable/cable in T.contents)
-		if(cable.d1 == 0 || cable.d2 == 0)
-			center_nodes++
-
-	if(center_nodes > 1)
-		Fail(T, "tile has multiple center cable nodes")
```

Comment:
accidental deletion here too?

Reply:
See above.

Reply:
Non-identical nodes can trigger this, for example 1 cable has `d1 = 1, d2 = 0` and a second cable has `d1 = 0, d2 = 4`.

Reply:
Ugh, what a pain, I can't express that in maplint, though maybe I can add it at some point (testing to see if both icon_states have a zero in them somewhere)